src/wallet/wallet.cpp

```diff
@@ -3299,6 +3299,9 @@ bool CWallet::InitLoadWallet()
             key.MakeNewKey(true);
             if (!walletInstance->SetHDMasterKey(key))
                 throw std::runtime_error("CWallet::GenerateNewKey(): Storing master key failed");
```

Comment:
Also the strings in the exceptions have the wrong function names. (Here as well as SetHDMasterKey())

Might as well  rework exceptions/return values for those?